src/main/java/com/google/javascript/clutz/ImportRenameMapBuilder.java

```diff
@@ -79,50 +88,71 @@ private String getGoogModuleId(Node astRoot) {
   }
 
   /**
-   * If we're running in incremental mode, closure gives imported symbols names in the current
-   * module's namespace, instead of the namespace of the module we're importing from. This reads the
-   * import statements to find the module id of the module that's being imported and builds a map
-   * from the names in the current module, to their correct counterparts.
+   * Does the actual work of walking over the AST, finding any goog.require() assignments or
```

Comment:
A good habit is starting comments on functions & methods with the name of the function, it sets the right voice for the sentence: `buildImportRenameMap does the actual...`.